migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -83,6 +83,7 @@ NFS does not require a defined storage class.
 If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-updating-deprecated-internal-images_troubleshooting-3-4[{product-title} 3 image stream tag] with `podman`.
 * [ ] The target cluster and the replication repository have sufficient storage space.
 * [ ] The xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+* [ ] Set the value of the `annotation.openshift.io/host.generated` parameter to `true` for each {product-title} route to update its host name for the target cluster. Otherwise, the migrated routes retain the source cluster host name.
```

Comment:
@sbeskin-redhat Will the user know how or where to set this parameter?

Reply:
@sbeskin-redhat  You did not quite answer Michael's question about "where" this is done. You need to say that it is in the route CR and mark it correctly.

Reply:
@apinnick  Actually, his question was not how and where to do it, but rather whether the user will know it without additional explanations. But an xref can be useful anyway.

Reply:
> @sbeskin-redhat Will the user know how or where to set this parameter?

@sbeskin-redhat  I don't think I misrepresented his question.